modules/ref/pages/client-settings.adoc

```diff
@@ -531,6 +531,21 @@ TIP: Most client operations that accept an options block allow for overriding th
 +
 See the advanced section in the documentation on more specific information on retry strategies and failure management.
 
+[#unordered-executions]
+Name: *Unordered Execution*::
+Builder Method: `unorderedExecutionEnabled(boolean)`
++
+Default:  _true_
++
+System Property: `com.couchbase.unorderedExecutionEnabled`
++
+From Couchbase 7.0, Out-of-Order returns prevent operations from being blocked by long-running KV requests, potentially improving performance for durable writes and multi-document ACID transactions.
+This means that tuning the number of connections (KV endpoints) is no longer necessary as a workaround where data not available in the cache is causing timeouts.
++
+This is set to `true` by default.
+Server will handle any operations that are not safe to be executed out of order, and setting this to `false` is unlikely to ever be necessary.
```

Comment:
Phrasing of this is maybe a little awkward?  E.g. I initially read as "Server will handle any operations that are not safe*,* to be executed out of order".  Maybe more clear just to omit this line?

Reply:
Yes, it isn't great - and it's more detail than I'd normally put on this page - owing to https://github.com/couchbase/docs-server/blob/relnotes-70/modules/release-notes/pages/relnotes.adoc#major-changes-in-behavior-700 needing to link here for more info.
Will comment out for now, and re-word if more information is asked for.